src/apps/FlappyBird.jsx

```diff
@@ -0,0 +1,362 @@
+import { useEffect } from "react";
+
+//=========================================
+
+const styles = {};
+styles.stage = {
+  display: "block",
+  maxWidth: "90vw",
+  borderRadius: "10px",
+  boxShadow: "0 0 20px rgb(0, 0, 0, 0.8)",
+};
+
+styles.stageContainer = {
+  display: "block",
+  width: "100%",
+  margin: "20px 0",
+};
+
+const NyanComponentRender = () => (
+  <div style={{ ...styles.stageContainer }} className="stageContainer">
+    <canvas
+      style={styles.stage}
+      className="stage"
+      id="stage"
+      width="800px"
+      height="600px"
+    />
+  </div>
```

Comment:
After the changes mentioned in the above comment are implemented, this can be changed to: 

```suggestion
  <StageContainer>
    <Stage
      id="stage"
      width="800px"
      height="600px"
    />
  </StageContainer>
```

Looks cleaner, don't you think?

Reply:
A lot better indeed. Nothing is better than being able to write css in its pure form. Thanks for the suggestion!

Reply:
If you use VS Code, remember to install their styled components extension so that css autocomplete works for styled components.

Reply:
Actually, I applied the changes you suggested directly from github. that caused the error. Thereafter, in another commit I applied the other changes and fixed that.

Reply:
No worries, I understand.